apps/web/lib/queries/projects/get-basic-project-info-by-slug.ts

```
@@ -84,6 +85,7 @@ export async function getBasicProjectInfoBySlug(
 
 	return {
 		id: project.id,
+		kindlerId: (project as any).kindler_id,
```

Biome (2.4.15)

[error] Unexpected any. Specify a different type.

(lint/suspicious/noExplicitAny)